doc/user_guide/electron_holography.rst

```diff
@@ -37,6 +37,8 @@ position and size as parameters:
 
 .. code-block:: python
 
+    >>> import hyperspy.api as hs
+    >>> im =  hs.datasets.example_signals.object_hologram()
     >>> wave_image = im.reconstruct_phase(sb_position=(<y>, <x>), sb_size=sb_radius)
 
```

Comment:
What about also adding some plotting lines and figures? This could help users with e.g. plotting the unwrapped phase.